kedro/framework/cli/starters.py

```diff
@@ -704,9 +704,14 @@ def _validate_config_file_against_prompts(
     if config is None:
         raise KedroCliError("Config file is empty.")
     missing_keys = set(prompts) - set(config)
-    if missing_keys:
+    if "project_name" in missing_keys:
         click.echo(yaml.dump(config, default_flow_style=False))
         raise KedroCliError(f"{', '.join(missing_keys)} not found in config file.")
+    if "example_pipeline" in missing_keys:
```

Comment:
Why are we checking only for `example_pipeline` and not `tools`? I think if `tools` will not be provided we also will use default values?

Reply:
Changing this!